bucket.go

```diff
@@ -112,19 +117,20 @@ func (client *Client) ListBuckets() ([]*Bucket, error) {
 
 // ListTestsInput represents the input to ListTests func
 type ListTestsInput struct {
-	BucketName string
-	Count      int
+	BucketKey string
```

Comment:
This is a breaking change, but I'm okay with that. We should start doing releases at some point